content/solana-nft.md

```diff
@@ -0,0 +1,618 @@
+
+### 2. Setup
+While in the project's root directory, lets first install the dependencies for our starter code:
+```sh
+npm install
+```
+
+Next, let's set up a new keypair to use for this lesson and airdrop some devnet SOL using our helper script. Run the following command:
+
+```sh
+npm start
+```
+You should now see a new `private-key.json` file. Copy the keypair from the new file and import it into Phantom.
+
+![Gif of Phantom Import](../assets/solana-nft-phantom.gif)
+
+### 3. Download Metaplex
+
+Now that our project has a new keypair, we're ready to set up the Metaplex helpers. While in the project's root directory, clone the Metaplex repository:
+
+```sh
+git clone https://github.com/metaplex-foundation/metaplex.git
+```
+
+This repository contains the Candy Machine v2 CLI tools. If you want to have a look at them more closely, you can find them in `metaplex/js/packages/cli/src/candy-machine-cli-v2.ts`.
+
+We'll be using `ts-nod` to run commands. If you don't have `ts-node` installed globally, install it now:
+
+```sh
+npm install -g typescript
+npm install -g ts-node
+```
+
+Next, install the Metaplex package dependencies:
+
+```sh
+yarn install --cwd metaplex/js/
+```
+
+Finally, let's confirm that everything is working by checking the Candy Machine v2 version:
+
+```sh
+ts-node metaplex/js/packages/cli/src/candy-machine-v2-cli.ts --version
+```
+
+If this command logs a version to the console, you should be good to go. If not, you'll have to debug the problem(s) before moving on. Check the logs if you're having issues.
+
+![Gif of Metaplex Install](../assets/solana-nft-metaplex-install.gif)
+
+### 4. Candy Machine Configuration
+
+Now that we have installed Metaplex, it's time to configure our candy machine.
+
+Open the `config.json` located in our metaplex-starter folder. Copy and paste the configurations below into `config.json`:
+
+```json
+{
+    "price": 1,
+    "number": 5,
+    "gatekeeper": null,
+    "solTreasuryAccount": "<YOUR_WALLET_ADDRESS>",
+    "splTokenAccount": null,
+    "splToken": null,
+    "goLiveDate": "25 Dec 2020 00:00:00 GMT",
+    "endSettings": null,
+    "whitelistMintSettings": null,
+    "hiddenSettings": null,
+    "storage": "arweave",
+    "ipfsInfuraProjectId": null,
+    "ipfsInfuraSecret": null,
+    "awsS3Bucket": null,
+    "nftStorageKey": null,
+    "noRetainAuthority": false,
+    "noMutable": false
+}
+```
+
+Update the `solTreasuryAccount` field with the wallet address we imported to Phantom. This will be the only setting we change for now.
+
+Let's walk through the values we have real quick:
+- `price` is set to `1`, meaning each NFT will cost 1 SOL.
```

Comment:
@jamesrp13 separate the list with a newline, ALSO: list items don't need ending punctuation

Reply:
I added punctuation to all of them because some of them are multiple sentences and have punctuation so I was keeping it consistent. Is that not needed? How should I handle the difference?

Reply:
@jamesrp13 for single sentence line items, I leave off the punctuation, but def agree, if the line item is more than one sentence, it feels better to have an ending punctuator.

imho, internal list consistency makes the most sense, but in general, unless needed (due to multiple sentences on one line), list items don't require ending punctuation (because the proposition is already delineated by being in the list).

Reply:
So if I remove all the trailing punctuation, would it be correct to remove the last period of the list item that has multiple sentences or to keep that period since it's now part of something more akin to a paragraph?

Reply:
@jamesrp13 I think it is okay to have an exception to the "no ending punc's in list items" rule

e.g. is okay when a line item w/many sentences ends w/a punctuator

More egregious than that would be a list w/inconsistent line endings, e.g.
1. Fred
2. Ted.
3. Bob
4. Alice
5. Jane.
6. John.
7. Jasmine